Simplenote/src/main/java/com/automattic/simplenote/utils/DialogUtils.java

```diff
@@ -0,0 +1,38 @@
+package com.automattic.simplenote.utils;
+
+import android.content.Context;
+import android.text.method.LinkMovementMethod;
+import android.widget.TextView;
+
+import androidx.appcompat.app.AlertDialog;
+import androidx.appcompat.view.ContextThemeWrapper;
+
+import com.automattic.simplenote.R;
+
+import java.util.Objects;
+
+public class DialogUtils {
+    /**
+     * Show an alert dialog with a link to the support@simplenote.com email address,
+     * which can be tapped to launch the device's email app.
+     *
+     * @param context   {@link Context} from which to determine theme and resources.
+     * @param title     {@link String} for the dialog title.
+     * @param message   {@link String} for the dialog message.
+     */
+    public static void showDialogWithEmail(Context context, String title, String message) {
```

Comment:
Star (🌟) Great job on extracting this duplicate logic from everywhere!

Idea (💡 ): Since the `title` is alway this `R.string.error` string, and based on the fact that you already decided to use the `R.string.support_email` within, which is currently only used for such errors, consider removing the `title` from the method arguments and hardcode it, just like you did for `R.string.support_email`. Then, the method can change to something like `showErrorDialogWithEmail(...)` instead.

Reply:
I had the `title` parameter so that future uses of the `showErrorDialogWithEmail` method could choose the title.  We could move `R.string.support_email` to a parameter too so that any email address could be used.  Since we're only using `R.string.error` for `title` at the moment, I removed the parameter in febd7eb4.